tiny_tds.gemspec

```diff
@@ -24,8 +24,8 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'mini_portile2', '~> 2.5.0'
   s.add_development_dependency 'rake', '~> 13.0.0'
   s.add_development_dependency 'rake-compiler', '~> 1.2'
-  s.add_development_dependency 'rake-compiler-dock', '~> 1.4.0'
-  s.add_development_dependency 'minitest', '~> 5.14.0'
+  s.add_development_dependency 'rake-compiler-dock', '~> 1.7.0'
+  s.add_development_dependency 'minitest', '~> 5'
```

Comment:
I think this syntax would allow updating major versions, which we probably wouldn't expect. I think `~> 5.25` would be safer